active/ImageIcon/ImageIconSpec.md

```diff
@@ -0,0 +1,143 @@
+    Image
+```
+
+# API Pages
+
+## ImageIcon class
+
+Represents an icon that uses an image as its content.
+
+ImageIcon can be used just like an
+[Image](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.Image)
+element, except it derives from
+[IconElement](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.IconElement)
+and can therefore be used anywhere an IconElement is required, such as
+[NavigationViewItem.Icon](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.NavigationViewItem.Icon).
+
+> Note: the ImageIcon.Foreground property is currently ignored.
+
+## Is this the right control? 
+
+Use an ImageIcon control when you want to display an image as your icon. Do not use an ImageIcon if you want to display an icon from a font. 
+
+## Examples
+Below is an example of how you can use ImageIcon in an App Bar Button Item.
+
+> Spec note: This example will also be added to the App Bar Button Item page with the rest of the icon controls. 
+
+XAML
+```xml
+<AppBarButton Label="ImageIcon" Click="AppBarButton_Click">
+    <AppBarButton.Icon>
+        <ImageIcon Source="Wheel.png"/>
+    </AppBarButton.Icon>
+</AppBarButton>
+```
+
+## ImageIcon member notes
+
+| Name	| Type | Description | Default |
+|:--- | :--- | :--- | :--- |
+|Source | ImageSource | Gets and sets the icon source file that will be used in the application. | Null |
+
+## ImageIconSource class
+
+_Same as above but with IconSource as the base type_
+
+# API Details
+
+```cs
+namespace Microsoft.UI.Xaml.Controls
+{
+    [webhosthidden]
+    unsealed runtimeclass ImageIcon : Windows.UI.Xaml.Controls.IconElement
+    {
+        ImageIcon();
+
+        Windows.UI.Xaml.Media.ImageSource Source{ get; set; };
+
+        static Windows.UI.Xaml.DependencyProperty SourceProperty{ get; };
+    }
+
+    [webhosthidden]
+    unsealed runtimeclass ImageIconSource : IconSource
+    {
+        ImageIconSource();
+
+        Windows.UI.Xaml.Media.ImageSource ImageSource{ get; set; };
+
+        static Windows.UI.Xaml.DependencyProperty ImageSourceProperty{ get; };
+    }
+}
+```
+
+# Appendix
+
+## Scope
+
+|Capability	| Priority |
+|:--- | :--- |
+| Icon can be used anywhere a BitmapIcon is accepted today | Must|
+| PNG, JPEG, and SVG images are types the control will accept | Must|
+| Icon must respond to user theming changes (Light, Dark, and High Contrast) | Could |
+| ICO icons will be accepted by the control | Could |
+| SVG image support will be improved | Won't |
+
+
+## Accessibility
+
+Since ImageIcon does not respond to the foreground property inherited from IconElement,
+it is up to the application to handle the icon color changes when the user’s theme is changed.  
+
+## Open Questions
+
+- Should ICOs be supported in the future? 
+- Should he ShowAsMonochrome property be added in the future? 
+This is somehing that was added because of a design requirement back when BitmapIcon was first designed but it is no longer a hard design requirement. 
```

Comment:
The long term goal is to try and sunset BitmapIcon and have ImageIcon replace it. The only property that is different between the two is the ShowAsMonochrom property. That property was added as it was in the design guidance at the time BitmapIcon was made to show icons as monochrome but that isn't always the case anymore. 

Your point about ShowAsMonochrome with SVG is a a good thing to call out because I am also not sure if that is something that is even feasible for SVG images.

Reply:
If we use the option @jevansaks  and @StephenLPeters  were thinking about using an effect, it could work for SVGs as well I think.

Reply:
This is a public API, not an internal Microsoft API, so we cannot assume that Contoso's design guidance is the same as Microsoft's internal design guidance. Contoso may want to maintain coherence with components that were designed in the Windows 8 era and which still use BitmapIcon, and the lack of monochrome support may be a migration issue for them. Not saying that we have to do it. Just that applying Microsoft internal policies to a public API may create friction.

Reply:
We have ideas how to do it but we don't have a customer scenario for it and so it's hard to justify doing the work without any demand. We could pre-approve the ShowAsMonochrome API from an API review perspective if someone ever came along and contributed the feature. :)